src/generated/resources/assets/c/lang/en_us.json

```diff
@@ -80,6 +80,10 @@
   "tag.block.c.ores_in_ground.stone": "Stone Ores In Ground",
   "tag.block.c.player_workstations.crafting_tables": "Crafting Tables",
   "tag.block.c.player_workstations.furnaces": "Furnaces",
+  "tag.block.c.pumpkins": "Pumpkins",
+  "tag.block.c.pumpkins.carved": "Carved Pumpkins",
+  "tag.block.c.pumpkins.jack_o_lanterns": "Jack O Lanterns",
```

Comment:
The translation should match the in-game's text which is Jack o'Lantern. So this should be `Jack o'Lanterns`
![image](https://github.com/user-attachments/assets/08b696e7-6b96-4282-bac5-eee1d1569529)